time,level,logger,message,callsite,exception
2020-05-03 20:13:41.0931,Error,Facelift_App.Repositories.CycleCountRepository,Validation failed for one or more entities. See 'EntityValidationErrors' property for more details.,Facelift_App.Repositories.CycleCountRepository.CreateAsync,"System.Data.Entity.Validation.DbEntityValidationException: Validation failed for one or more entities. See 'EntityValidationErrors' property for more details.
   at System.Data.Entity.Internal.InternalContext.SaveChangesAsync(CancellationToken cancellationToken)
   at System.Data.Entity.Internal.LazyInternalContext.SaveChangesAsync(CancellationToken cancellationToken)
   at System.Data.Entity.DbContext.SaveChangesAsync(CancellationToken cancellationToken)
   at System.Data.Entity.DbContext.SaveChangesAsync()
   at Facelift_App.Repositories.CycleCountRepository.<CreateAsync>d__3.MoveNext() in D:\Duta Kalingga Pratama\Projects\Unilever\Facelift\Application\UNILEVER_Facelift\Facelift_App\Repositories\CycleCountRepository.cs:line 130"
2020-05-03 20:14:42.7714,Error,Facelift_App.Repositories.CycleCountRepository,Validation failed for one or more entities. See 'EntityValidationErrors' property for more details.,Facelift_App.Repositories.CycleCountRepository.CreateAsync,"System.Data.Entity.Validation.DbEntityValidationException: Validation failed for one or more entities. See 'EntityValidationErrors' property for more details.
   at System.Data.Entity.Internal.InternalContext.SaveChangesAsync(CancellationToken cancellationToken)
   at System.Data.Entity.Internal.LazyInternalContext.SaveChangesAsync(CancellationToken cancellationToken)
   at System.Data.Entity.DbContext.SaveChangesAsync(CancellationToken cancellationToken)
   at System.Data.Entity.DbContext.SaveChangesAsync()
   at Facelift_App.Repositories.CycleCountRepository.<CreateAsync>d__3.MoveNext() in D:\Duta Kalingga Pratama\Projects\Unilever\Facelift\Application\UNILEVER_Facelift\Facelift_App\Repositories\CycleCountRepository.cs:line 130"
